src/components/Title.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+
+Title.propTypes = {
+  pageName: PropTypes.string,
```

Comment:
I'd say a `h1` without a text does not make sense. You could add `.isRequired` here.